packages/sled/index.js

```diff
@@ -61,6 +61,28 @@ function getNestedHandler(handlerString) {
   }
 }
 
+/**
+ * Given an inputs and the file name of a schema, validates the input.
+ * @param {String} input The input, config, or output to be validated
+ * @param {String} schemaFile The relative path to the schema
+ * @returns {Promise} A Promise resolving to true if Input is valid or rejecting if it errors
+ */
+function validateMessage(input, schemaFile) {
+  if (input && schemaFile) {
+    return readJsonFile(schemaFile)
+      .then((schema) => {
+        const ajv = new Ajv();
+        const schemaValidator = ajv.compile(schema);
+        const valid = schemaValidator(input);
+        if (!valid) {
+          throw Error('Validation Error', schemaValidator.errors);
+        }
+        return valid;
```

Comment:
`return true;` would make it more clear that this is returning a boolean.  (The ajv docs don't make it totally clear what "valid" is set to if there were no errors.)